src/frontegg-mock/src/lib.rs

```diff
@@ -1425,12 +1616,58 @@ pub struct ApiToken {
     #[serde(alias = "client_id")]
     pub client_id: Uuid,
     pub secret: Uuid,
+    #[serde(default)]
+    pub description: Option<String>,
+    #[serde(alias = "created_at", skip_deserializing)]
+    pub created_at: DateTime<Utc>,
+}
+
+#[derive(Serialize, Deserialize, Clone)]
+pub struct UserApiTokenResponse {
+    #[serde(rename = "clientId")]
+    pub client_id: String,
+    pub description: String,
+    #[serde(rename = "created_at")]
+    pub created_at: DateTime<Utc>,
+    pub secret: String,
+}
+
+#[derive(Deserialize)]
+pub struct UserApiTokenRequest {
+    pub description: String,
```

Comment:
Should this be an `Option`? In the docs, it seems that the description is only required in the response. It's a weird API, but we should probably follow it.

We also don't have the optional `metadata` and `expiresInMinutes` fields. That's probably ok, if we just haven't implemented them.